infection.json.dist

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "timeout": 30,
```

Comment:
Let's lower the timeout to 5 seconds or such: we don't have long-running integration tests anyway

Reply:
With default 10 seconds a lot mutations were skipped due to timeout. But I‘ll try to reduce it in the dist.

Reply:
Done. Many skipped tests now which reduces the MSI. They are not considered T(imed out).

Reply:
from the quick look, `S`kipped mutants should not decrease MSI:

![image](https://user-images.githubusercontent.com/3725595/145197998-0b5127c4-1d60-4364-a541-5130d29a031e.png)

but if you found a case when it does, please report

Reply:
To clarify: Skipped tests do not reduce MSI but are not accounted for in the metric at all.

I guess there are quite a few integration tests which do take time combined with possible mutation .. more than expected.

While the phpunit testsuite takes around 18-20s, mutation testing for the same test suite takes around 9m (if timeout is set to 30 and no S(kipped) mutants emerge).

Reply:
Doesn't that reduce the coverage itself?

Reply:
Yes, but tests should be isolated to a certain unit :D 

Mutation testing integration tests is expensive, so we need to decide how coverage leaks in there.

Reply:
Possible way to move forward:
- accept the 30s timeout for now
- in subsequent issue add `@covers` And `@group integration|unit`
- infection and coverage only on `@group unit` so that coverage/MSI does not decrease

I volunteer to write additional tests ;-)

Reply:
> I volunteer to write additional tests ;-)

Cool, your problem now :D

Reply:
> Cool, your problem now :D

Not in this PR maybe please? ;-)